content/en/containers/docker/_index.md

```diff
@@ -281,6 +281,10 @@ Returns `CRITICAL` if the Agent is unable to connect to Datadog, otherwise retur
 **datadog.agent.check_status**: <br>
 Returns `CRITICAL` if an Agent check is unable to send metrics to Datadog, otherwise returns `OK`.
 
+## Uninstall Single Step APM Instrumentation
+
+If you used Single Step APM Instrumentation to install the Datadog Docker Agent, and you want to uninstall the Agent, you need to [run additional commands][33] to uninstall Single Step Instrumentation.
```

Comment:
change 
1. "to uninstall Single Step Instrumentation." to "to uninstall APM Instrumentation." 
2. "If you used Single Step APM Instrumentation to install the Datadog Docker Agent," to "If you installed the Datadog Docker Agent with Single Step APM Instrumentation"

Reply:
2. is relevant for Linux Agents too.